dasharo-security/secure-boot.robot

```diff
@@ -129,281 +121,210 @@ SBO003.001 Attempt to boot file with the correct key from Shell (firmware)
     ...    converted to DER using `openssl base64 -d -in DB-ECDSA521.cer -out DB-ECDSA521.der`
-    Mount ISO As USB In QEMU    ${CURDIR}/../scripts/secure-boot/images/ECDSA521.img
+    Skip If    not ${SECURE_BOOT_SUPPORT}    SBO010.006 not supported
+    Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    SBO010.006 not supported
+    Mount ISO As USB    ${CURDIR}/../scripts/secure-boot/images/ECDSA521.img
     Power On
 
     ${sb_menu}=    Enter Secure Boot Menu And Return Construction
     Enable Secure Boot    ${sb_menu}
-    ${sb_menu}=    Reenter Menu And Return Construction
-    ${advanced_menu}=    Enter Advanced Secure Boot Keys Management And Return Construction    ${sb_menu}
-    Enter Enroll DB Signature Using File In DB Options    ${advanced_menu}
-    Enter Volume In File Explorer    ECDSA521
-    Select File In File Explorer    cert.der
-    Save Changes And Reset    3    5
+    ${key_menu}=    Enter Key Management And Return Construction
+    Enroll DB Signature    ${key_menu}    ECDSA521    cert.der
+    Save Changes And Reset
 
-    Enter UEFI Shell
-    ${out}=    Execute File In UEFI Shell    signed-hello.efi
-    Should Contain    ${out}    Hello, world!
+    Boot Efi File    signed-hello.efi    ECDSA521    Hello, world!
 
 SBO011.001 Attempt to enroll expired certificate and boot signed image
     [Documentation]    This test verifies that an expired certificate can be
     ...    used to verify booted image.
     Skip If    not ${SECURE_BOOT_SUPPORT}    SBO011.001 not supported
     Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    SBO011.001 not supported
-    Skip If    not ${TESTS_IN_UBUNTU_SUPPORT}    SBO011.001 not supported
-    Mount ISO As USB In QEMU    ${CURDIR}/../scripts/secure-boot/images/EXPIRED.img
+    Mount ISO As USB    ${CURDIR}/../scripts/secure-boot/images/EXPIRED.img
     Power On
     ${sb_menu}=    Enter Secure Boot Menu And Return Construction
     Enable Secure Boot    ${sb_menu}
-    Save Changes
-    Reenter Menu
-    ${sb_menu}=    Get Secure Boot Menu Construction
-    ${advanced_menu}=    Enter Advanced Secure Boot Keys Management And Return Construction    ${sb_menu}
-    Enter Enroll DB Signature Using File In DB Options    ${advanced_menu}
-    Enter Volume In File Explorer    EXPIRED
-    Select File In File Explorer    cert.der
-    Save Changes And Reset    3    5
-    Enter UEFI Shell
-    ${out}=    Execute File In UEFI Shell    hello.efi
-    Should Contain    ${out}    Access Denied
-    ${out}=    Execute File In UEFI Shell    signed-hello.efi
-    Should Contain    ${out}    Access Denied
+    ${key_menu}=    Enter Key Management And Return Construction
+    Enroll DB Signature    ${key_menu}    EXPIRED    cert.der
+    Save Changes And Reset
+
+    Boot Efi File Should Fail    hello.efi    EXPIRED
+    Reset System
+    Boot Efi File Should Fail    signed-hello.efi    EXPIRED
 
 SBO012.001 Boot OS Signed And Enrolled From Inside System (Ubuntu 22.04)
```

Comment:
@m-iwanicki Faced following error while running tests on commit `236b638a06073132378566a98d62c37195c7c83e`:

```bash
SBO012.001 Boot OS Signed And Enrolled From Inside System (Ubuntu ... | FAIL |
'False' should be true.
------------------------------------------------------------------------------
```

Reply:
This fail was caused by not installed `sbctl` because of problem in QEMU configuration, no fix needed.